react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -403,6 +403,8 @@ class RCTAztecView: Aztec.TextView {
             let end = selection["end"]  as? NSNumber {
             setSelection(start: start, end: end)
         }
+        // This signal the RN/JS system that the component needs to the relayout
```

Comment:
Small typo (I think)
```suggestion
        // This signals the RN/JS system that the component needs to relayout
```